### PR TITLE
Submit handler for reservation_options. BBS-91

### PR DESCRIPTION
**Pull request by tripox**
reservation_options_submit

### CODE REVIEW

File: modules/aleph/includes/aleph.reservation.inc [hunk 1]
```diff
@@ -16,6 +16,32 @@ function aleph_reservation_options() {
...
+
+  // Use a metadata wrapper to access the data.
+  $wrapper = entity_metadata_wrapper('profile2', $profile);
+
+  // Get values.
+  $preferred_branch = $wrapper->field_aleph_preferred_branch->value();
+
+  // If default branch was not found in the profile, update the profile with the
+  // one selected.
+  if (!$preferred_branch) {
```
Comment (xendk) [inline modules/aleph/includes/aleph.reservation.inc:38]: How about if preferred branch != the submitted?

**Review by xendk — APPROVED** [review]
Minor thing.

File: modules/aleph/includes/aleph.reservation.inc (hunk 1 above)
Comment (kasperg) [reply modules/aleph/includes/aleph.reservation.inc:38]: That would in effect change the preferred branch for the user every time they select another branch. I do not think that is appropriate.

**Review by kasperg — APPROVED** [review]
:+1: LGTM

**Pull request merged**